<robot
  name="rokae">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-0.0247702979705072 8.83157337442717E-06 -0.0913242486824033"
        rpy="0 0 0" />
      <mass
        value="5.92495775488409" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="Link_1">
    <inertial>
      <origin
        xyz="0.0117373446679598 -0.00012100659180864 0.083722431531897"
        rpy="0 0 0" />
      <mass
        value="3.47774247106277" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_1"
    type="continuous">
    <origin
      xyz="0 0 0"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link_1" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="Link_2">
    <inertial>
      <origin
        xyz="0.0297425061950912 -0.000782130246424844 0.273103924765273"
        rpy="0 0 0" />
      <mass
        value="3.60654764202097" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_2"
    type="continuous">
    <origin
      xyz="0 0 0"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link_2" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="Link_3">
    <inertial>
      <origin
        xyz="0.0472501501211913 -0.00163562155508445 0.44069909563997"
        rpy="0 0 0" />
      <mass
        value="1.77260685945223" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_3"
    type="continuous">
    <origin
      xyz="0 0 0"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link_3" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="Link_4">
    <inertial>
      <origin
        xyz="0.215375169114832 -0.00104054798236882 0.455620232292236"
        rpy="0 0 0" />
      <mass
        value="1.71885043928238" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_4"
    type="continuous">
    <origin
      xyz="0 0 0"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link_4" />
    <axis
      xyz="1 0 0" />
  </joint>
  <link
    name="Link_5">
    <inertial>
      <origin
        xyz="0.339826606744663 -0.00227290936047979 0.454490393526931"
        rpy="0 0 0" />
      <mass
        value="0.373336262982894" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_5.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_5.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_5"
    type="continuous">
    <origin
      xyz="0 0 0"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link_5" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="Link_6">
    <inertial>
      <origin
        xyz="0.395741723708409 -0.000333318400527269 0.454361780437817"
        rpy="0 0 0" />
      <mass
        value="0.00385662836396193" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_6.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://rokae/meshes/Link_6.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="Joint_6"
    type="continuous">
    <origin
      xyz="0 0 0"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="Link_6" />
    <axis
      xyz="1 0 0" />
  </joint>
</robot>